presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -576,6 +577,11 @@ public SystemSessionProperties(
                         OMIT_DATETIME_TYPE_PRECISION,
                         "Omit precision when rendering datetime type names with default precision",
                         featuresConfig.isOmitDateTimeTypePrecision(),
+                        false),
+                booleanProperty(
+                        USE_LEGACY_WINDOW_FILTER_PUSHDOWN,
+                        "Use legacy window filter pushdown optimizer",
+                        true,
```

Comment:
Toggles for Optimizer are usually defined in `FeaturesConfig`. Please add it there, and default to `false`.